github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -975,88 +975,7 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
-  - name: pull-kubevirt-e2e-k8s-1.17-rook-ceph-nonroot
```

Comment:
There was a need for ceph-NonRoot and now it is missing. Is this on purpose or an omission?

Reply:
I'll put it back and use 1.20 for this lane instead - probably safer. It's unclear what the nonroot lanes are about.